src/components/buttons/SecondaryLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+type Props = {
+  to: string;
+  children: JSX.Element[] | JSX.Element;
+};
+
+export const SecondaryLink: React.FC<Props> = ({ to, children }) => {
```

Comment:
To takich linków z X można zrobić, nie lepiej jeden generyczny komponent który przyjmuje zmienną "variant" której wartościami jest unia możliwych wariantów?